specifyweb/frontend/js_src/lib/components/FormSliders/RecordSet.tsx

```diff
@@ -283,24 +297,52 @@ function RecordSet<SCHEMA extends AnySchema>({
       })
     );
 
+  async function createNewRecordSet(
+    ids: RA<number | undefined>
+  ): Promise<void> {
+    await recordSet.save();
+    await Promise.all(
+      ids.map(async (resource) =>
+        createResource('RecordSetItem', {
+          recordId: resource,
+          recordSet: recordSet.get('resource_uri'),
+        })
+      )
+    );
+    navigate(`/specify/record-set/${recordSet.id}/`);
+  }
+
   return (
     <>
       <RecordSelectorFromIds<SCHEMA>
         {...rest}
+        createNewRecordSet={createNewRecordSet}
         defaultIndex={currentIndex}
         dialog={dialog}
-        headerButtons={<EditRecordSetButton recordSet={recordSet} />}
+        headerButtons={
+          recordSet.isNew() && ids.length > 0 && !currentRecord.isNew() ? (
```

Comment:
Is it possible to have the`<Slider />` and the create record set button appear only after the total count is more than one (for new record sets only)?

This way the slider won't be visible if you are creating just one resource (so it will look like just like it was before this pull request) but as soon as the user clicks on `Add`, the slider will re-appear